aiohttp/helpers.py

```diff
@@ -592,6 +593,20 @@ def weakref_handle(
     return None
 
 
+def create_eager_task(
```

Comment:
Personally, I'd rather duplicate such code through the codebase, it makes cleaning it up in future easier.
e.g. If we just run pyupgrade or whatever on this, it's likely to result in a pointless-looking wrapper:
```
def create_eager_task(coro, loop):
    return asyncio.Task(coro, loop=loop, eager_start=True)
```

Whereas, running on duplicated code, we'd just end up with the expected code. For that same reason, I'd only add a comment explaining why we want eager_start. The availability of the feature on certain Python versions is obvious from the code itself.

Reply:
I'll undo the de-duplication.  I think we only have 4-5 potential places where this makes sense anyways.